cat-l1b.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash -u
 
 # keeping track of where I am
-DIR_NOW=$( cd $(dirname $0); pwd )
+DIR_NOW="$(cd $(dirname $BASH_SOURCE); pwd )"
+DIR_NOW=$(pwd)
```

Comment:
I think this will only work if you are in the same directory as this script. Can you please check?

Reply:
I think both will only work in the local folder 
Here is what I got if I run that line
 ```
$ DIR_NOW=$(cd $(dirname $BASH_SOURCE); pwd)
dirname: missing operand
Try 'dirname --help' for more information.

```
I guess because $0 and $BASH_SOURCE does not return the proper directory in my case.
```
$ echo $0
-bash
$ echo $BASH_SOURCE


```

Reply:
Sorry, you are right. 

I tested before and ` DIR_NOW=$(cd $(dirname $BASH_SOURCE); pwd )` did not work on the terminal window.
```
$ echo $(cd $(dirname $BASH_SOURCE); pwd)
dirname: missing operand
Try 'dirname --help' for more information.
```
I checked it again. It works well within the script. really interesting. 
We should keep it as what it is

Reply:
The BASH_SOURCE variable is only defined inside a script, cf. https://stackoverflow.com/a/35006505/2047215